src/lib/server/account.js

```diff
@@ -159,4 +159,23 @@ export async function getArchiveParts(userId) {
 	values: [userId]
 	});
 	return rows;
+}
+
+export async function softDeleteUser(pool, uuid) {
```

Comment:
Pourquoi prendre `pool` en paramètres ? La variable peut être exportée et elle l'est déjà.
```suggestion
export async function softDeleteUser(uuid) {
```